rocksdb/README.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Copyright (c) 2012 - 2017 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on RocksDB running locally.
```

Comment:
does it, though?

Reply:
Yes I believe so. Why do you think otherwise?

Reply:
I see no installation and setup instructions for RocksDB itself :/ As I understood it, these are also in the binding-related readme.md

Reply:
You don't need to install it. Everything you need is self-contained inside the rocksdbjni Jar that YCSB declares a dependency on.